src/measure_theory/integral/integrable_on.lean

```diff
@@ -246,6 +246,21 @@ begin
   exact ((Lp.mem_ℒp _).restrict s).mem_ℒp_of_exponent_le hp,
 end
 
+lemma lintegral_to_real_le_lintegral_nnnorm (f : α → ℝ) :
```

Comment:
That lemma should be moved to an earlier file since it is not about set integrals.